### ISSUE TITLE
LastPass Cannot Generate Secure Password

### ISSUE THREAD

**Issue by jonathansampson** [brave/browser-laptop#4115]
**Did you search for similar issues before submitting this one?**

Yes.

**Describe the issue you encountered:**

LastPass is unable to generate a new secure password.

**Expected behavior:**

LastPass is able to generate a new secure password.
- Platform (Win7, 8, 10? macOS? Linux distro?): Windows 10
- Brave Version: 0.12.1
- Steps to reproduce:
  1. Open LastPass Popup
  2. Click "Generate Secure Password"
  3. Note that nothing is generated
- Screenshot if needed:
  ![image](https://cloud.githubusercontent.com/assets/815158/18642742/be47728e-7e56-11e6-82fc-36bf96f230a2.png)
- Any related issues:

To my knowledge, none that are related to password-generation, or script-execution within the context of LastPass.

**Comment by Instinct64** [comment]
I've just recently started experiencing this issue on (Windows 10 x64):

Brave	0.12.15
Muon	1.4.31
libchromiumcontent	53.0.2785.143
V8	5.3.332.47
Node.js	6.5.0
Update Channel	dev
os.platform	win32
os.release	10.0.14393
os.arch	x64

**Comment by jonathansampson** [comment]
@Instinct64 Can you test this in 0.13.0?

**Comment by Instinct64** [comment]
@jonathansampson I still see the issue in 0.13.1

**Comment by jonathansampson** [comment]
@Instinct64 Which version of Brave and LastPass are you presently using? The full Brave version details can be loaded in `about:brave`, and the LastPass version can be found in `about:extensions`.

**Comment by Instinct64** [comment]
@jonathansampson Apologies for the delayed response. This issue still occurs in the latest release:

Brave	0.13.4
rev	71d8ffc
Muon	2.56.4
libchromiumcontent	56.0.2924.87
V8	5.6.326.50
Node.js	7.4.0
Update Channel	dev
os.platform	win32
os.release	10.0.14393
os.arch	x64

LastPass: Free Password Manager  4.1.38.67

**Comment by jonathansampson** [comment]
@Instinct64 You and I have identical settings, yet mine appears to be working. Would you be available to Skype or jump into a Google Hangout, and briefly look at your instance together?

**Comment by Instinct64** [comment]
@jonathansampson I'd be happy to. Add me on Skype: "imchrisbrooks"

**Comment by jonathansampson** [comment]
I spoke with @Instinct64 yesterday at length about this. We were unable to determine the exact cause of the issue. No exceptions were being thrown by any script. It _may_ be the case that LastPass simply wasn't loading some of their scripts.

We found that removing the `%appdata%/brave` folder, and re-installing LastPass, resolved the problem.

I'm closing this issue, but will keep an ear open for anybody else experiencing similar problems.

**Comment by Ryan-Myers** [comment]
I've been experiencing this issue as an ongoing issue for months on Firefox. This is across multiple computers that are using Firefox sync together. Uninstalling and reinstalling Lastpass will temporarily resolve the issue, however, it will return the next day (possibly after using another computer). 

I'm also not seeing any exceptions being thrown in the console, and seemingly nothing works. I can't generate passwords when registering and attempting to use the automatic fill in pop-up. It also won't work when trying to generate a password manually through the dropdown when clicking on the icon in the toolbar.

**Comment by luixxiul** [comment]
@Ryan-Myers please consult with Firefox customer support. This repository is for Brave, not Firefox, thanks!

**Comment by Ryan-Myers** [comment]
I'm sorry! I completely spaced out when reporting this issue, and didn't realize this wasn't a LastPass specific bug report. I came here from Google. I apologize.

**Comment by luixxiul** [comment]
@Ryan-Myers nevermind! if you don't mind, please try Brave once ;-)

**Comment by JasonSelkirk** [comment]
This might sound completely idiotic and people have probably tested this already but I thought I would share just in case as I came here looking for answers when I noticed the Advanced Options drop-down on the password generation tab of LastPass. I had never clicked it before but I did and noticed that neither of the radio boxes Allow All Character Type or Make Pronounceable were ticked. I just ticked the first one and the password appeared. I login to chrome on two machines and one had this ticked and one lost the tick.
